test/journeys/lib/test-helpers/roster.js

```diff
@@ -0,0 +1,21 @@
+import {assert} from 'chai';
```

Comment:
My first attempt at a "page object" style helper. Hopefully will reduce some copy-pasta in our journey tests.